modules/nw-sriov-app-netutil.adoc

```diff
@@ -18,7 +18,7 @@ Currently there are three API methods implemented:
 +
 [NOTE]
 ====
-Exposing hugepages via Kubernetes Downward API is an alpha feature in Kubernetes 1.20 and is not enabled in {product-title}. The API can be tested by enabling the feature gate, `FEATURE_GATES="DownwardAPIHugePages=true"` on Kubernetes 1.20 or greater.
+Exposing hugepages via Kubernetes Downward API is an alpha feature in Kubernetes 1.21 and is not enabled in {product-title}. The API can be tested by enabling the feature gate, `FEATURE_GATES="DownwardAPIHugePages=true"` on Kubernetes 1.21 or greater.
```

Comment:
@jboxman Can you check whether these updates from 1.20 -> 1.21 are appropriate or not for 4.8?

Reply:
@Billy99, what do you think? Thanks!

Reply:
This is already being handled by #30619.
The FeatureGate moved to beta (still disabled upstream) and is now enabled by SR-IOV Operator. This NOTE is being removed.

Reply:
Thank you both! I'll revert this change.